src/batched/KokkosBatched_Vector_SIMD_Arith.hpp

```diff
@@ -0,0 +1,725 @@
+#ifndef __KOKKOSBATCHED_VECTOR_SIMD_ARITH_HPP__
+#define __KOKKOSBATCHED_VECTOR_SIMD_ARITH_HPP__
+
+/// \author Kyungjoo Kim (kyukim@sandia.gov)
+
+#include "Kokkos_Complex.hpp"
+
+namespace KokkosBatched {
+  namespace Experimental {
+
+    //#define SIMD_ARITH_RETURN_TYPE(A) typename std::enable_if<std::is_same<Kokkos::Impl::ActiveExecutionMemorySpace,Kokkos::HostSpace>::value,A >::type
+#define SIMD_ARITH_RETURN_TYPE(T,l) Vector< SIMD< T >, l >
```

Comment:
Please namespace macros with e.g., `KOKKOS_KERNELS_`.

Reply:
I got it.